vim-mode.js

```diff
@@ -177,19 +177,27 @@ function copyUrlToClipboard () {
 // We should provide means to disable the keybings for a blacklist of websites
 // github.com for example has a lot of vim-like bindings of its own.
 
+// When using the "j" or "k" Vim commands. (j = down, k = up)
+const scrollAmount = 60
+// When using the "d" or "u" Vim commands. (d = down, u = up)
+const quickScrollAmount = 400
+
 // We put scrolling here in keydown, so that we can continously press it.
 document.addEventListener('keydown', function (e) {
-  if ((e.key === 'j' || e.key === 'k') && !isLinkKeyMode && !isCurrentlyInInput()) {
+  if ((e.key === 'j' || e.key === 'k' || e.key === 'd' || e.key === 'u') && !isLinkKeyMode && !isCurrentlyInInput()) {
     if (e.key === 'j') {
-      window.scrollBy(0, 60)
+      window.scrollBy(0, scrollAmount)
     } else if (e.key === 'k') {
-      window.scrollBy(0, -60)
+      window.scrollBy(0, -scrollAmount)
+    } else if (e.key === 'd') {
+      window.scrollBy(0, quickScrollAmount)
+    } else if (e.key === 'u') {
+      window.scrollBy(0, -quickScrollAmount)
     }
   }
 })
 
 const commandChars = new Set(['f', 'F', 'y', 'g', 'G', 'c'])
-const linkChars = new Set(alphabet)
```

Comment:
Why is this deleted? This gets used on line 251.

Reply:
That's an oversight and out of scope diff. My b